### PR TITLE
Default builds:output to the latest build when no id is specified

### LINKED ISSUE #47: Have `build:outputs` default to last build
Can we have

```
⛄  2.4.1 🚀  ~/documents/projects/codetriage (master)
$ heroku builds:output -a issuetriage
```

Default to the same thing as running

```
$ heroku builds
```

Grabbing the first ID and then passing it in to

```
$ heroku builds:output <ID> -a issuetriage
```

### PR DESCRIPTION
**Pull request by dmathieu**
Closes #47

### CODE REVIEW

File: commands/builds/output.js [hunk 1]
```diff
@@ -12,21 +13,41 @@ module.exports = {
...
+      partial: true,
+      headers: { 'Range': 'updated_at ..; max=1, order=desc' }
+    }).then((builds) => builds[0])
+  }
+  let build
+
+  if (context.args.id) {
+    build = yield heroku.get(`/apps/${context.app}/builds/${context.args.id}`)
+    if (!build) {
+      cli.error('No build found with that app and uuid')
```
Comment (tt) [inline commands/builds/output.js]: "ID" instead of "uuid"

**Comment by dmathieu** [comment]
I'll be rebasing this PR against #50 before merging it (and have it use the helpers introduced there).

**Comment by schneems** [comment]
Thanks! Do we have to manually update to get changes on plugins?

**Comment by dmathieu** [comment]
I haven't shipped a new version yet. I'll do that soon. Then it should auto-update.

**Pull request merged**